### PR TITLE
Update Finaid deadline that is now open

### PR DESCRIPTION
**Pull request by estyxx**

### CODE REVIEW

**Review by thepetk — APPROVED** [review]
lgtm

**Comment by estyxx** [comment]
I cannot merge this one (if you are waiting for me...) 😄

**Comment by thepetk** [comment]
> I cannot merge this one (if you are waiting for me...) 😄

Hey! I was just waiting for any further reviewers :) Merged!

**Pull request merged**
